packages/preset-icons/src/types.ts

```diff
@@ -55,6 +55,20 @@ export interface IconsOptions {
    */
   warn?: boolean
 
+  /**
+   * Safe `@iconify-json` collections to use (will be also auto installed when missing and `autoInstall` enabled).
+   *
+   * This options should be used only when there are new `@iconify-json` collections not listed in the default icons preset collection names.
+   *
+   * Adding external collections will not work, you should use `FileSystemIconLoader` from
+   * `@iconify/utils/lib/loader/fs` or `createExternalPackageIconLoader` from
+   * `@iconify/utils/lib/loader/external-pkg` instead.
+   *
+   * @see https://unocss.dev/presets/icons#filesystemiconloader
+   * @see https://unocss.dev/presets/icons#externalpackageiconloader
+   */
+  safeCollectionsNames?: string[]
```

Comment:
```suggestion
  iconifyCollectionsNames?: string[]
```

In this context, I think `safe` is a bit ambiguous and confusing.

Reply:
looks like npm registry is down

Reply:
I'm going to add the new option to the docs